src/coprocessor_v2/plugin_registry.rs

```diff
@@ -278,6 +278,10 @@ struct PluginRegistryInner {
     /// Plugins that are currently loaded.
     /// Provides a mapping from the plugin's name to the actual instance.
     loaded_plugins: HashMap<String, (OsString, Arc<LoadedPlugin>)>,
+
+    /// Paths of loaded *and* unloaded plugins.
+    /// Files in this list should not be loaded again.
+    library_paths: HashSet<OsString>,
```

Comment:
Since no plugin will be unloaded, what does this differ from the `loaded_plugins`?

Reply:
Plugins will not be unloaded but they can be renamed when the corresponding `so` file being renamed. But I believe that it is the *original* path that should not be reload again.  (I can't reproduce the TLS related behaviour in my computer though)

Reply:
The doc comment is a bit outdated, I will update it soon.

Reply:
Maybe we can make rename a noop either.

Reply:
fair enough